test/test_helper.rb

```diff
@@ -62,6 +62,10 @@ def set_basic_auth(user, password)
     @request.env['HTTP_ACCEPT'] = 'application/json'
   end
 
+  def fetch_proxy_features
+    ProxyAPI::Features.any_instance.stubs(:features => Feature.name_map.keys)
```

Comment:
The previous one was for unit (/models) tests. This one is for functional (/controllers) tests. I guess I can remove it too and add per each test.